index.html

```diff
@@ -178,21 +178,39 @@ <h2>About Me</h2>
               I'm always interested in hearing about new projects, so if you'd
               like to chat please get in touch.
             </p>
```

Comment:
- [ ] Please notice that in the original figma design the font-size property for this paragraph is 32px, you have set it to 28px currently, as one of the main goal of the project is to parse a figma template as close to as possible, please fix the font-size propert
currently: 
```
#contact p {
  font-size: 28px;
}
```

Reply:
@PinkMoon25 Thank you for pointing those out

Reply:
@adamilare you got it 👍